frontend/model/contracts/chatroom.js

```diff
@@ -457,11 +458,11 @@ sbp('chelonia/defineContract', {
     'gi.contracts/chatroom/deleteMessage': {
       validate: actionRequireInnerSignature((data, { state, message: { innerSigningContractID }, contractID }) => {
         objectOf({
-          hash: string,
+          hash: stringMax(MAX_HASH_LEN, 'hash'),
           // NOTE: manifestCids of the attachments which belong to the message
           //       if the message is deleted, those attachments should be deleted too
           manifestCids: arrayOf(string),
-          messageSender: string
+          messageSender: stringMax(MAX_HASH_LEN, 'messageSender')
```

Comment:
Just above this is `arrayOf(string)`. Can you modify that to be `arrayOf(stringMax(MAX_HASH_LEN, 'manifestCids'))`?